engine/events/magikarp.asm

```diff
@@ -108,36 +108,18 @@ CalcMagikarpLength:
 ;   de: wEnemyMonDVs
 ;   bc: wPlayerID
 
-; This function is poorly commented.
-
-; In short, it generates a value between 190 and 1786 using
-; a Magikarp's DVs and its trainer ID. This value is further
-; filtered in LoadEnemyMon to make longer Magikarp even rarer.
+; It generates a value between 190 and 1755 using a Magikarp's DVs
+; and its trainer ID. This value is further filtered in LoadEnemyMon
+; to make longer Magikarp even rarer.
```

Comment:
```suggestion
; It generates a length (in mm) between 190 and 1755 using a Magikarp's DVs
; and its Trainer ID.
```
The comment about being further filtered in `LoadEnemyMon` assumes this function is only called from `LoadEnemyMon` (ignoring the case when it is called from `CheckMagikarpLength`). IMO, what a calling function does with the return value is out of the scope of what belongs in comments on this function.